docs/native/src/testing/testcases.rst

```diff
@@ -1,3 +1,140 @@
-==============================
+Please download the :download:`Testing <ManualTestPass.docx>` document and fill it in as you perform your testing. After both the validator and manual tests are both 100% passing, please provide Microsoft with a link to your validator app, your test account information and manual test results document. Microsoft will complete a final test pass before launch. 
+
+Please contact Microsoft for directions to set the Office for iOS apps into a test mode that can be used for testing the below scenarios. 
+
+
+ ..  note::
+	 * "Company app/application" refers to your app. 
+	 * "Company service" refers to your service. 
+	 * Where it refers to "Office", please substitute Word, Excel and PowerPoint app. The tests should be re-run against each Office application. 
+	 * Please note the version of the Office app you tested against. This information can be found under Settings -> [App] -> Version
+
+.. |Duplicate| image:: ../images/PartnerTestCases_Duplicate.png  
+    :alt: A screenshot that shows the document actions in Office for iOS. 
+.. |ImageProps1| image:: ../images/PartnerTestCases_Properties1.png 
+    :alt: A screenshot that shows document properties in the Office outspace. 
+.. |ImageProps2| image:: ../images/PartnerTestCases_Properties2.png
+    :alt: A screenshot that shows the document properties within the opened file. 
+
+-------------------------------------
+1 - Open from Company App - First Install
+-------------------------------------
+*This test verifies the flow of using Office for the first time from the Company app. Repeat for each supported file type and each Office app.*
+
+#. Start with a fresh install of Company app. Ensure Office is not installed.
+#. Boot up company app and login.
+#. **RESULT: General promotion for Office should be shown the first time after upgrading the company app.**
+#. Browse to a supported file type
+#. **RESULT: "Open with Microsoft [app]" promotion, drawing attention to control and enabling open with Office once per first open for each supported file type. "Open with Microsoft should be top choice if multiple choices are available. If a list is not shown Office should be the default app for opening the file.**
+#. Activate control to open in Office
+#. **RESULT: User should be sent to app store page for the corresponding app.**
+#. Install the Office app.
+#. After installation, go back to the Company app and activate the control to open in Office again.
+#. **RESULT: Office should start. User should be prompted for credentials to Company Service.**
+#. Enter credentials for company service.
+#. **RESULT: File should open in read only mode.**
+#. Click sign in and sign in with a free Microsoft Account.
+#. **RESULT: File should open in edit mode if user is a consumer and read-only mode if the user is a commercial user.**
+#. Make changes [you will need to sign in with a subscription account for testing commercial user]
+#. Click Back (<-)
```

Comment:
Use [guilabel](http://www.sphinx-doc.org/en/stable/markup/inline.html#role-guilabel) whenever you're referring to a piece of UI